### ISSUE TITLE
`dash_core_components` is deprecated

### ISSUE THREAD

**Issue by hoangphucITJP** [man-group/dtale#576]
Hi. I'm running `dtale.show` and it produces some warnings. Not a severe issue, but you can take a look.

```
The dash_core_components package is deprecated. Please replace
`import dash_core_components as dcc` with `from dash import dcc`

The dash_html_components package is deprecated. Please replace
`import dash_html_components as html` with `from dash import html`

The dash_html_components package is deprecated. Please replace
`import dash_html_components as html` with `from dash import html`
```

Configuration:
```
dash==2.2.0
dtale==1.57.0
```

**Comment by aschonfeld** [comment]
@hoangphucITJP thanks for the update! I have updated this and will include it in the next release.

There is still once instance of the warning coming from `dash_bootstrap_components` but it looks like they are getting ready to release a new version using the new import style.

**Comment by aschonfeld** [comment]
Fixed in v1.58.1
